utils/text_formatting.jsx

```diff
@@ -326,8 +326,13 @@ function highlightCurrentMentions(text, tokens, mentionKeys = []) {
             flags += 'i';
         }
 
-        const pattern = new RegExp(`(^|\\W)(${escapeRegex(mention.key)})(\\b|_+\\b)`, flags);
-
+        let pattern;
+        if (cjkPattern.test(mention.key)) {
+            // In the case of CJK mention key, even if there no delimiters (such as spaces) at both ends of a word, it is recognized as a mention key
```

Comment:
Good to change but non-blocking
```suggestion
            // In the case of CJK mention key, even if there's no delimiters (such as spaces) at both ends of a word, it is recognized as a mention key
```